modules/conflict_resolver/test/TestPlan.md

```diff
@@ -44,3 +44,27 @@
  - Also check and make sure that the instrument and its corresponding DDE are updated accordingly
  - Ensure if Date_taken is changed, the candidate age in the instrument is updated accordingly
  - Make sure the scoring_fields are re-calculated in the given instrument once the conflict is resolved
+
+## Dashboard Widget
+
+1. Go to the LORIS dashboard. 
+ - Ensure that the "My Tasks" "My Tasks" on the LORIS dashboard has a "Data entry conflicts"
+   item
+ - Ensure that the `access_all_profiles` permission changes the label from "Sites: All"
+   to "Sites: User sites" and the number associated with the task is updated accordingly.
```

Comment:
```suggestion
   and "Sites: User sites" and the number associated with the tasks is updated accordingly.
```

Reply:
Ok for "task" to "tasks" but the "to" is Ok.

Reply:
I think it was supposed to be a multiline suggest, now that the line above it is changed it should be "and"